evap/evaluation/models.py

```diff
@@ -263,6 +263,18 @@ def is_single_result(self):
 
         return self.contributions.get(responsible=True).questionnaires.filter(name_en=Questionnaire.SINGLE_RESULT_QUESTIONNAIRE_NAME).exists()
 
+    def num_single_result_participants(self):
+        if not self.is_single_result:
+            return 0
+
+        num_participants = 0
+        ratinganswercounters = RatingAnswerCounter.objects.filter(
+            question=Question.objects.get(questionnaire=Questionnaire.get_single_result_questionnaire()),
+            contribution=self.contributions.get(contributor=self.responsible_contributor))
+        for counter in ratinganswercounters:
+            num_participants += counter.count
+        return num_participants
```

Comment:
this should be a private method and be called by num_voters and num_participants if self.is_single_result.

actually, i think we should put this directly into the _participant_count and _voter_count variable. unfortunately, those variables is currently used for determining whether a course is archived, which is used to determine whether a semester is archived. we could allow semesters with archived and non-archived courses at the same time, but a semester with only one course which is a single result would still be archived then.

i think this might require an is_archived flag on Semester and/or Course to to correctly. lets discuss that tomorrow.